Documentation/admin/ingress.md

```diff
@@ -150,8 +150,16 @@ metadata:
 
 When rules are defined and the Ingress Controller is running, traffic to your Service will be routed appropriately.
 
+### HTTP header limitations
+
+Using reverse proxies that limit HTTP headers to 4K may result in errors for large group counts. If using HTTP headers for large cookie replacement, set the 
```

Comment:
Perhaps something more like: 

`By default, Ingress limits HTTP headers to 8KB.  This limit may cause problems with applications that send very large headers, such as Tectonic Console when users are members of a very large number of groups. To increase the limit, set the flag...'